ERR_PNPM_PEER_DEP_ISSUES  Unmet peer dependencies

After upgrading to @codspeed/vitest-plugin 5.0.0, the peerDependencies error occured:

```
 ERR_PNPM_PEER_DEP_ISSUES  Unmet peer dependencies

.
└─┬ @codspeed/vitest-plugin 5.0.0
  └── ✕ unmet peer tinybench@^2.9.0: found 5.0.1
```

This does not match the description from the [release note](https://github.com/CodSpeedHQ/codspeed-node/releases/tag/v5.0.0):

> 💥 Breaking Changes
>
> tinybench: Support for versions below v4 has been dropped. You must upgrade to tinybench v4 or above.
> ⚠️ Since tinybench v4 drops support for CJS, ES modules are now required to use the library.

See https://github.com/lynx-family/lynx-stack/pull/1863.

--- a/packages/tinybench-plugin/package.json
+++ b/packages/tinybench-plugin/package.json
@@ -35,6 +35,6 @@
     "stack-trace": "1.0.0-pre2"
   },
   "peerDependencies": {
-    "tinybench": "^4.0.1"
+    "tinybench": ">=4.0.1"
   }
 }

--- a/packages/vitest-plugin/package.json
+++ b/packages/vitest-plugin/package.json
@@ -31,7 +31,7 @@
     "@codspeed/core": "workspace:^5.0.0"
   },
   "peerDependencies": {
-    "tinybench": "^2.9.0",
+    "tinybench": ">=2.9.0",
     "vite": "^4.2.0 || ^5.0.0 || ^6.0.0 || ^7.0.0",
     "vitest": ">=3.2"
   },

